### ISSUE TITLE
Pouring Dynamics Estimation Using Gated Recurrent Units. (arXiv:2105.12828v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7355]
Pouring Dynamics Estimation Using Gated Recurrent Units. (arXiv:2105.12828v1 [cs.LG])<br>
https://ift.tt/3oWOqXy<br>
<p>One of the most commonly performed manipulation in a human's daily life is pouring. Many factors have an effect on target accuracy, including pouring velocity, rotation angle, geometric of the source, and the receiving containers. This paper presents an approach to increase the repeatability and accuracy of the robotic manipulator by estimating the change in the amount of water of the pouring cup to a sequence of pouring actions using multiple layers of the deep recurrent neural network, especially gated recurrent units (GRU). The proposed GRU model achieved a validation mean squared error as low as 1e-4 (lbf) for the predicted value of weight f(t). This paper contains a comprehensive evaluation and analysis of numerous experiments with various designs of recurrent neural networks and hyperparameters fine-tuning.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
